docs/export-format.md

```diff
@@ -0,0 +1,158 @@
+# Briefcase export CSV format
+
+## Format parameters
+
+| Parameter               | Value | Description |
+| ----------------------- | ----- | ----------- |
+| Separator               | `,`   | Comma character |
+| Quoted                  | `"`   | Double quote character. See the [Values block columns](#values-block-columns) section to know which fields get quoted |
+| Header                  | Yes   | Every file will have a header with the respective column names |
+| Empty columns           | Yes   | See the "Values block columns" section to know which fields get empty columns |
+| Encoding of null values | No    | Nulls are just empty columns and, sometimes, empty strings |
+
+## Main submission output file
+
+The main submission out file has values from any non repeating group field found at the topmost level of a form's model and other metadata.
+
+### Structure of the file:
+
+| Column block | Description |
+| ------------ | ----------- |
+| Fixed block  | Holds a column for the submission date |
+| Values block | Holds columns with the values of topmost model fields and non repeating groups |
+| Fixed block  | Holds a column for the submission key and, when the form is encrypted, a content validation flag |
+
+### Fixed block columns
+
+| Column name    | Optional | Quoted | Description |
```

Comment:
For optional, I was expecting the inverse. That is, I want to know what things are Required.